### PR TITLE
Add a few links and Twitter profiles

### PR DESCRIPTION
**Pull request by kmuncie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,20 @@
   * [Paciello Group](https://developer.paciellogroup.com/resources/)
   * [Ebay accessibility patterns](https://ebay.gitbooks.io/mindpatterns/content/)
   * [BBC accessibility guidelines](http://www.bbc.co.uk/guidelines/futuremedia/accessibility/)
+  * [Vox Media Accessibility Checklist](http://accessibility.voxmedia.com/)
 
-## Blogs, etc.
+## Other Resources (Blogs, Individuals, Videos, etc)
```
Comment (acemuncie) [inline README.md]: Good change! I think we can actually get rid of the parentheses and what's within them, to make it more concise.

File: README.md [hunk 2]
```diff
@@ -13,13 +13,20 @@
...
 
   * [24 Accessibility](https://www.24a11y.com/) - series of articles on digital accessibility
-  * [Leonie Watson](https://tink.uk/)
-  * [Scott O'Hara](https://www.scottohara.me/all-writings/)
+  * [Leonie Watson](https://tink.uk/) - Leading Accessibility engineer, on W3C Web Platform Working Group. [@LeonieWatson](https://twitter.com/LeonieWatson)
```
Comment (acemuncie) [inline README.md]: To be consistent, can we lowercase `Leading`, and remove the period at the end?

File: README.md [hunk 3]
```diff
@@ -13,13 +13,20 @@
...
+  * [Vox Media Accessibility Checklist](http://accessibility.voxmedia.com/)
 
-## Blogs, etc.
+## Other Resources (Blogs, Individuals, Videos, etc)
 
   * [24 Accessibility](https://www.24a11y.com/) - series of articles on digital accessibility
-  * [Leonie Watson](https://tink.uk/)
-  * [Scott O'Hara](https://www.scottohara.me/all-writings/)
+  * [Leonie Watson](https://tink.uk/) - Leading Accessibility engineer, on W3C Web Platform Working Group. [@LeonieWatson](https://twitter.com/LeonieWatson) 
+  * [Scott O'Hara](https://www.scottohara.me/all-writings/) UX Accessibility at Paciello Group. [@scottohara](https://twitter.com/scottohara)
```
Comment (acemuncie) [inline README.md]: Also for consistency - add hyphen after the link, lowercase `Accessibility`, and remove period

File: README.md [hunk 4]
```diff
@@ -13,13 +13,20 @@
...
   * [Alastair Campbell](https://alastairc.ac/)
   * [Inclusive Components](https://inclusive-components.design/) - pattern library of accessible web interfaces
-  * [A11ycasts with Rob Dodson](https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g) - series of videos on the fundamentals of building accessible apps by Rob Dodson, developer advocate at Google
+  * [Video: A11ycasts with Rob Dodson](https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g) - series of videos on the fundamentals of building accessible apps by Rob Dodson, developer advocate at Google
```
Comment (acemuncie) [inline README.md]: Let's make Video its own heading, just like *Helpful Topical Reads*, then move Leonie Watson's video under that. We can remove the *Video* label from the links.

**Review by acemuncie — CHANGES_REQUESTED** [review]
Fantastic resources, thanks! Just a few comments to address.

**Comment by kmuncie** [comment]
Back to you @acemuncie

**Pull request merged**